opengrok-indexer/src/main/jflex/analysis/fortran/FortranXref.lex

```diff
@@ -86,13 +86,14 @@ File = [a-zA-Z]{FNameChar}* ".inc"
     onFilteredSymbolMatched(yytext(), yychar, Consts.kwd, false);
 }
 
+"'" ({File}|{FPath}) "'" |
 "<" ({File}|{FPath}) ">" {
     chkLOC();
-    onNonSymbolMatched("<", yychar);
-    String file = yytext();
-    file = file.substring(1, file.length() - 1);
+    String cmatch = yytext();
+    onNonSymbolMatched(cmatch, yychar);
```

Comment:
the 1st argument should be the first character of `cnmatch`, not the whole string, I believe.